### ISSUE TITLE
Contribute Back to Hyrax/Hyku

### ISSUE THREAD

**Issue by ShanaLMoore** [notch8/utk-hyku#489]
# Story

TODO: Find all accessibility related tickets for UTK and contribute back to hyku

- https://github.com/scientist-softserv/utk-hyku/issues/40
- PR: https://github.com/scientist-softserv/utk-hyku/pull/488

# Acceptance Criteria
- [ ] Work should be contributed back to proper source

# Notes
  - [x] https://github.com/scientist-softserv/utk-hyku/issues/485
    - [x] #41 (had a needs rework label so will need to look at the following PRs to determine final product)
      - [x] https://github.com/scientist-softserv/utk-hyku/pull/519
      - [x] https://github.com/scientist-softserv/utk-hyku/pull/493
  - [x] #43
    - [x] https://github.com/scientist-softserv/utk-hyku/issues/471 (had a needs rework label so will need to look at the following PRs to determine final product)
      - [x] https://github.com/scientist-softserv/utk-hyku/pull/505
      - [x] https://github.com/scientist-softserv/utk-hyku/pull/469

**Comment by jillpe** [comment]
To Do:
- [Filter on accessibilit](https://github.com/orgs/scientist-softserv/projects/43/views/1?filterQuery=label%3AAccessibility)y and locate all of the PR's
- attach them as bullet points to this PR
- This is the work we want to contribute back to hyku
